R/assess_downloads.r

```diff
@@ -0,0 +1,40 @@
+#' Assess a package for the number of downloads in the past year
+#' 
+#' @details The more times a package has been downloaded the more extensive the user testing and the greater chance there is of someone finding a bug and logging it.
+#'
+#' @eval assess_family_roxygen(
+#'   "downloads_1yr",
+#'   "a numeric value between [0,1] indicating the volume of downloads")
+#'
+#' @export
+assess_downloads_1yr <- function(x, ...){
+  UseMethod("assess_downloads_1yr")
+}
+# assign a friendly name for assess column
+attr(assess_downloads_1yr,"column_name") <- "downloads_1yr"
+attr(assess_downloads_1yr,"label") <- "number of downloads in the past year"
+
+pkg_ref_cache.downloads_1yr <- function(x, name, ...) {
+  cran_downloads(x$name, from=Sys.Date()-365, to=Sys.Date()) %>% group_by(package) %>% summarize(downloads_1yr = sum(count))
```

Comment:
I think it's a good idea to cache the whole `data.frame` that is returned by `cran_downloads` since it might be useful for other download-related metrics - for example, if we have a "downloads_1yr" and "downloads_1mo", both can use the same data without downloading it twice. 

```r
pkg_ref_cache.downloads_1yr <- function(x, ...) {
  cran_downloads(x$name, from=Sys.Date()-365, to=Sys.Date()) 
}
```